streamer/src/streamer.rs

```diff
@@ -267,13 +267,17 @@ pub fn recv_vec_packet_batches(
         .iter()
         .map(|packets| packets.packets.len())
         .sum::<usize>();
-    while let Ok(packet_batch) = recvr.try_recv() {
-        trace!("got more packets");
-        num_packets += packet_batch
-            .iter()
-            .map(|packets| packets.packets.len())
-            .sum::<usize>();
-        packet_batches.extend(packet_batch);
+    while num_packets < 100_000 {
```

Comment:
Would it make sense to make this a configurable with a reasonable default value?